api/extractors/extractor.h

```diff
@@ -1,10 +1,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include "ccextractor.h"
+#include "../../src/lib_ccx/ccx_common_common.h"
```

Comment:
You don't need to specify the path, that's what's the include dirs are for.  "ccx_common_common.h" is enough.